schema/swagger.yml

```diff
@@ -2334,7 +2334,11 @@ definitions:
       currentTime:
         type: integer
         example: 1533558858128
-        description: Current time of the node (Unix Time Stamp).
+        description: Current time of the node in miliseconds (Unix Timestamp).
+      epochTime:
```

Comment:
It may have some usability on third party. But to avoid any confusion we should rename this attribute to `secondsSinceEpoch`, because when ever we talk about `epochTime` it always referred to a constant value specified to a network.